lib/android/app/src/reactNative71/java/com/reactnativenavigation/react/ReactGateway.java

```diff
@@ -35,17 +36,15 @@ public void onActivityResumed(NavigationActivity activity) {
 	}
 
     public boolean onNewIntent(Intent intent) {
-        if (host.hasInstance()) {
-            host.getReactInstanceManager().onNewIntent(intent);
+        if (host.getCurrentReactContext() != null) {
+            host.onNewIntent(intent);
             return true;
         }
         return false;
     }
 
     public void onConfigurationChanged(NavigationActivity activity, @NonNull Configuration newConfig) {
-        if (host.hasInstance()) {
-            host.getReactInstanceManager().onConfigurationChanged(activity, newConfig);
-        }
+        host.onConfigurationChanged(activity);
```

Comment:
Any sense in adding this safety check? -
```suggestion
        if (host.getCurrentReactContext() != null) {
            host.onConfigurationChanged(activity);
        }
```